pkg/chunk/chunk_store.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/cortexproject/cortex/pkg/util/validation"
 )
 
+// Exported errors.
```

Comment:
Lint complaining again? Otherwise this comment is just noise.

Reply:
My local editor complains, I'll remove this.